src/views/Invite/Cohort.js

```diff
@@ -181,7 +181,7 @@ export default function InviteCohort() {
         <Grid.Item full>
           <Flex align="center">
             <Flex.Item>
-              {_acceptedInvites + _pendingInvites} / {_totalInvites}
```

Comment:
Huh, I thought this was correct, in that these are the amount of invites you've actually _used_. idk though.